Resources/Prototypes/_Moffstation/PersonalItems/Items/LillianMothette/Lilith_Mothette/lilithguitar.yml

```
@@ -0,0 +1,31 @@
+﻿# Player: lilly_the_ratty - Character: Lilith Mothette
+- type: entity
+  parent: ElectricGuitarInstrument
+  id: PersonalItemLilithGuitar
+  name: heavily used guitar
+  description: This guitar looks like it has seen better days and has a lot of suspicious stickers on it
+  suffix: PersonalItem
+  components:
+  - type: Sprite
+    sprite: _Moffstation/PersonalItems/Items/LillianMothette/Lilith_Mothette/lilithguitar.rsi
+    state: icon
+  - type: Item
+    sprite: _Moffstation/PersonalItems/Items/LillianMothette/Lilith_Mothette/lilithguitar.rsi
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - back
+    - suitStorage
+    sprite: _Moffstation/PersonalItems/Items/LillianMothette/Lilith_Mothette/lilithguitar.rsi
+  - type: DetailExaminable
+    content: On the back of the guitar there is an engraving of the name Lilith Mothette. All over it there is crayon marks that seem to be left by a child. There are scratch marks all around the srings.  One of the strings also is a completely different one making it look like a cheap replacement. The head of the guitar is also taped on and somehow still holding. On the base there are two stickers; one you find familiar, but cant put your finger on it. The other has the letters LT in a bright pink.
```

Reword for clarity, its quite hard to read in its current state.

For physical descriptions, going from most obvious to least obvious (unique) is a good way to lead into a question.
```suggestion
    content: The guitar is covered in crayon marks that seem to be left by a child. The strings are heavily scratched, with one string looking like a cheap replacement. The head is taped on and somehow still holding. On the base, there are two stickers; one you find familiar, but can't put your finger on it. The other has the letters LT in a bright pink. On the back of the body is an engraving of the name Lilith Mothette.
```